.github/workflows/ci.yml

```diff
@@ -1,16 +1,38 @@
 name: CI
 
 on:
+  workflow_dispatch:
   pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
   push:
-    branches: main
+    branches:
+      - main
+      - nightly
   schedule:
-    - cron: "0 18 * * 1,4,6" # 1800 UTC every Monday, Thursday, Saturday
+    - cron: "30 18 * * 1,4,6" # 1830 UTC every Monday, Thursday, Saturday
```

Comment:
Not totally related to original purpose, but moving this 30 minutes later to ensure that we pick up c2pa-rs nightly which also runs at 1800 UTC.